cli/cmd/deploy/deploy.go

```diff
@@ -293,6 +294,13 @@ func DeployFlow(ctx context.Context, ch *cmdutil.Helper, opts *Options) error {
 		return fmt.Errorf("create project failed with error %w", err)
 	}
 
+	err = dotrillcloud.SetAll(localProjectPath, ch.AdminURL, &dotrillcloud.RillCloud{
+		ProjectID: res.Project.Id,
+	})
+	if err != nil {
+		return err
+	}
```

Comment:
This might not be safe. I believe `localProjectPath` can be empty when deploying a Github remote directly. Maybe need to check `isLocalGitPath` before calling this?